### ISSUE TITLE
Streamline emoji reactions UI

### ISSUE THREAD

**Issue by joshproehl** [vector-im/element-android#3517]
#### Is your feature request related to a problem? Please describe.
When trying to deal with emoji reactions to a message I have to consider multiple UI elements at different places.

After long-pressing a message, the popup contains the following non-contiguous UI elements regarding reactions:
* The quick-reaction emoji bar
* The Add Reaction menu item
* The View Reactions menu item

This causes an (admittedly tiny) bit of cognitive overhead due to having to read the entire UI to figure out how to add a non-quick emoji, and then adding a little extra confusion because two non-contiguous menu items have an icon that represents a reaction/emoji.

#### Describe the solution you'd like.
Ideally I'd like to have the last item on the quick-bar be a (...) indicating that I can tap that to select from the wider list of emojis, and remove the redundant "Add Reaction" menu item.

#### Describe alternatives you've considered. 
Instead of removing the "add reaction" menu item it could be moved down so that the me menu item order is:
* Copy
* Quote
* Add Reaction
* View Reactions

**Comment by bmarty** [comment]
Yes, I think you are right, we could at least group "Add reaction" and "View Reactions".

This is what we have so far:

<img width="323" alt="image" src="https://user-images.githubusercontent.com/3940906/122613809-c0df0080-d085-11eb-8214-1ee6a85e58c7.png">

Let's ask the design team for a bigger reordering.

**Comment by github-tomster** [comment]
@ free-time-app-creators
if there is indeed a 'design team'
▶️ https://github.com/vector-im/element-android/pull/2514#issuecomment-788757748
▶️ https://github.com/vector-im/element-android/issues/2944
▶️ https://github.com/vector-im/element-android/pull/3182#issuecomment-840877213
